modules/core/src/lib/deck-picker.js

```diff
@@ -220,7 +214,7 @@ export default class DeckPicker {
           pass: `picking:${mode}`,
           redrawReason: 'pick-z',
           pickZ: true
-        });
+        })?.pickedColors;
```

Comment:
Nit: Slapping that at the end of a multiline function call is hard to read IMO. I would break in two lines with intermediate variable.